src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-size: 100px;
+  color: #fff;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  width: 300px;
+  height: 300px;
+  flex-grow: 1;
+  flex-shrink: 0;
+}
+
+.block-1 {
+  order: 1;
+  background-color: #f00;
+}
+
+.block-2 {
+  order: 2;
+  background-color: #c00;
+}
+
+.block-3 {
+  order: 3;
+  background-color: #900;
+}
+
+.block-4 {
+  order: 4;
+  background-color: #600;
+}
+
+.block-5 {
+  order: 5;
+  background-color: #300;
+}
+
+.block-6 {
+  order: 6;
+  background-color: #000;
+}
+
+@media (min-width: 600px) {
+  .block-3 {
+    order: 4;
+  }
+
+  .block-4 {
+    order: 3;
+  }
+}
+
+@media (min-width: 900px) {
+  .container {
+    max-width: 1199px;
```

Comment:
don't restrict container width, it should be 3 columns on all resolutions > 1200 and also it should be fullscreen, not like that:
![image](https://user-images.githubusercontent.com/37161315/154504090-504fc8a7-b3ae-4b69-9d0f-6a909dcd0c99.png)